packages/optimizer/lib/transformers/MinifyHtml.js

```diff
@@ -77,8 +75,11 @@ class MinifyHtml {
     if (opts.canCollapseWhitespace && !this.canCollapseWhitespace(node.tagName)) {
       childOpts.canCollapseWhitespace = false;
     }
-    if (opts.canTrimWhitespace && !this.canTrimWhitespace(node.tagName)) {
-      childOpts.canTrimWhitespace = false;
+    if (node.tagName === 'head' || node.tagName === 'html') {
+      childOpts.inBody = false;
+    }
+    if (node.tagName === 'body') {
```

Comment:
This should be an `else if`, this is mutually exclusive to the previous `if`.

Reply:
done